WordPress/src/main/java/org/wordpress/android/util/EventBusAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package org.wordpress.android.util
+
+import de.greenrobot.event.EventBus
+import javax.inject.Inject
+
+/**
+ * Provides an interface for [de.greenrobot.eventbus.EventBus] which can be mocked and used in unit tests.
+ */
+class EventBusAdapter @Inject constructor() {
```

Comment:
Since I knew this was technically an Adapter, I wanted to avoid `*Wrapper` since it's overused that it means a lot of things now. But you're right about it being confused with Android's `Adapter` class. I changed it in d4568f4588399f2286ddddca710af1d43b1f18e0. 😞